core/src/components/button/button.ionic.scss

```diff
@@ -0,0 +1,239 @@
+@import "./button";
+@import "./button.ionic.vars";
+
+// Ionic Button
+// -------------------------------------------------------------------------------
+
+:host {
+  --border-radius: #{$button-ionic-default-border-radius};
+  --padding-top: #{$button-ionic-default-padding-top};
+  --padding-end: #{$button-ionic-default-padding-end};
+  --padding-bottom: var(--padding-top);
+  --padding-start: var(--padding-end);
+
+  position: relative;
+
+  min-height: #{$button-ionic-default-min-height};
+
+  font-size: #{$button-ionic-default-font-size};
+
+  // Target area
+  &::after {
+    position: absolute;
```

Comment:
Ionic Framework includes a set of Sass mixins in favor of directional properties that can change from LTR → RTL. 

Here, we should use the `@include position(50%, 0, 0, 0);` mixin. It will align with code standards in other style files and remove the need for disabling stylelint.